src/stores/pizza.store.ts

```diff
@@ -0,0 +1,198 @@
+import axios from 'axios';
+import { defineStore, storeToRefs } from 'pinia';
+import { computed, ref } from 'vue';
+import type { Order } from '@/models/order';
+import type { Pizza } from '@/models/pizza';
+import type { AdminTimeslotDeref, Export, Timeslot } from '@/models/timeslot';
+
+import { useUserStore } from './user.store';
+
+const { get_csrf } = useUserStore();
+const { csrf } = storeToRefs(useUserStore());
+
+export const usePizzaStore = defineStore('pizza', () => {
+  const pizzaList = ref<Record<number, Pizza>>({});
+  const timeslotList = ref<Record<number, Timeslot | AdminTimeslotDeref>>({});
+
+  const pizzas = computed(() => Object.values(pizzaList.value));
```

Comment:
I don't see any where this 'pizzas' computed value is used, could we remove it ?

Reply:
You're right, this is same as pizzaList but unused. I deleted it.